### PR TITLE
Add package config for bower (fixes #70).

### LINKED ISSUE #70: Support installing RemoteStorage modules through Bower
In order to make it easier to re-use common modules, it would be nice if we could install them through Bower as well. Preferably a package per module, but with the list of modules this small it would already be a useful improvement if all of them could be installed and managed at once using Bower.

### PR DESCRIPTION
**Pull request by Vinnl**
Based on the contents of package.json.

### CODE REVIEW

**Comment by Vinnl** [comment]
Note that if this PR is accepted, it won't be listed on http://bower.io/search/ until [bower register remotestorage-modules https://github.com/remotestorage/modules.git](http://bower.io/docs/creating-packages/#register) is ran.

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "remotestorage-modules",
+  "description": "remotestorage modules collection",
+  "keywords": "remotestorage modules collection",
+  "main": "src/*.js",
+  "ignore": [
+  ],
+  "version": "0.0.1",
+  "private": false,
+  "author": "Niklas Cathor <nilclass@riseup.net>",
```
Comment (skddc) [inline bower.json]: Unfortunately, @nilclass has given up on all public open-source work some time ago, and these modules are written by a whole bunch of people. I think the key supports an array, right (or maybe that was just npm)? Maybe we should just list all names from the contributors page and do the same for remotestorage.js.

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "license": "AGPL",
```
Comment (skddc) [inline bower.json]: All rs.js projects are licensed under MIT these days. We removed the MIT/GPL double licensing 2 or so years ago in favor of the more permissive one, as it is compatible with GPL anyway.

File: bower.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  "homepage": "http://github.com/remotestorage/modules",
+  "dependencies": {
+    "remotestorage": "^0.10.2",
```
Comment (skddc) [inline bower.json]: Current stable is `0.11.0`.

**Comment by skddc** [comment]
Cool thing! Added some comments.

File: bower.json (hunk 1 above)
Comment (Vinnl) [inline bower.json]: Bower doesn't even define an author field, I just left it in as it probably wouldn't really do harm. The information in package.json even used incorrect syntax -- I've fixed that now. According to [the docs](https://docs.npmjs.com/files/package.json), the author is a single person (i.e. the person who started the project), and you can add a contributors array for other people. I've declined doing that for now as that would require looking up everybody's email address and having them consent with publishing it here.

File: bower.json (hunk 2 above)
Comment (Vinnl) [inline bower.json]: Updated in both bower.json and package.json.

File: bower.json (hunk 3 above)
Comment (Vinnl) [inline bower.json]: Updated in both bower.json and package.json.

**Comment by Vinnl** [comment]
I've updated the pull request to incorporate your comments :)

**Comment by skddc** [comment]
+1

**Comment by galfert** [comment]
+1

**Comment by skddc** [comment]
Thanks, @Vinnl!

**Comment by Vinnl** [comment]
Nice, thanks for merging :)

**Pull request merged**